xml/System/String.xml

```diff
@@ -13296,13 +13296,13 @@ The following example defines a `StripStartTags` method that uses the <xref:Syst
  :::code language="fsharp" source="~/snippets/fsharp/System/String/Substring/source.fs" id="Snippet1":::
  :::code language="vb" source="~/snippets/visualbasic/VS_Snippets_CLR_Classic/classic String.Substring1 Example/VB/source.vb" id="Snippet1":::
 
- The following example uses the <xref:System.String.Substring%2A> method to separate key/value pairs that are delimited by an equals ("=") character.
+ The following example uses the <xref:System.String.Substring%2A> method to separate key/value pairs that are delimited by an equals (`"="`) character.
```

Comment:
```suggestion
 The following example uses the <xref:System.String.Substring%2A> method to separate key/value pairs that are delimited by an equals (`=`) character.
```

Reply:
I’ve also replaced two more instances of `"="`.